edx_sga/__init__.py

```diff
@@ -2,5 +2,4 @@
 Module for StaffGradedAssignmentXBlock.
 """
 
-__version__ = '0.11.1'
-
+__version__ = '0.12.0'
```

Comment:
@mraarif Doof will do the versioning for us so do not need it in the PR.

Reply:
reverted